ios/Classes/PTFlutterViewController.h

```diff
@@ -15,19 +15,18 @@ NS_ASSUME_NONNULL_BEGIN
 @property (nonatomic) BOOL documentLoaded;
 
 // viewer options
-@property (nonatomic, assign, getter=isReadOnly, setter=setReadOnly:) BOOL readOnly;
-@property (nonatomic, assign, getter=isThumbnailViewEditingEnabled, setter=setThumbnailViewEditingEnabled:) BOOL thumbnailViewEditingEnabled;
-@property (nonatomic, copy, getter=getAnnotationAuthor) NSString * annotationAuthor;
-@property (nonatomic, assign, getter=isContinuousAnnotationEditing, setter=setContinuousAnnotationEditing:) BOOL continuousAnnotationEditing;
-
-- (void)setAnnotationAuthor:(NSString*)annotationAuthor;
+@property (nonatomic, assign) BOOL readOnly;
+@property (nonatomic, assign) BOOL thumbnailViewEditingEnabled;
```

Comment:
This property doesn't look correctly implemented to me, using only the default accessor methods.